web_widget_x2many_2d_matrix/README.rst

```diff
@@ -50,15 +53,25 @@ field_label_y_axis
     Use another field to display in the table header
 field_value
     Show this field as value
+field_editability
+    Use this field to evaluate if a cell should be editable or not. It
```

Comment:
finish the sentence, probably explaining that this is supposed to be truthy if editable, and not truthy otherwise. How about a serialized field that returns a dict with properties such as `readonly` and `invisible`?